test/distributions/test_distributions.py

```diff
@@ -2272,14 +2275,18 @@ def gradcheck_func(samples, nu, sigma, prec, scale_tril):
         wishart_log_prob_gradcheck(df_no_batch, None, None, scale_tril_batched)
 
     def test_wishart_stable_with_precision_matrix(self):
-        x = torch.randn(10)
+        ndim = 10
+        x = torch.randn(ndim)
         P = torch.exp(-(x - x.unsqueeze(-1)) ** 2)  # RBF kernel
-        Wishart(torch.tensor(10), precision_matrix=P)
+        Wishart(torch.tensor(ndim), precision_matrix=P)
 
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_wishart_log_prob(self):
-        df = (torch.rand([], requires_grad=True) + 1) * 10
-        tmp = torch.randn(3, 10)
+        ndim = 3
+        df = torch.rand([], requires_grad=True) + ndim - 1
+        if version.parse(scipy.__version__) < version.parse("1.7.0"):
+            df += 1.
```

Comment:
Same as the other comment - could you expand on this and maybe add a comment as to what changed in this version number?

Reply:
That would be great. I will add comments here. Thank you for the suggestion.